packages/actions/src/ImportAction.php

```diff
@@ -0,0 +1,373 @@
+<?php
+
+namespace Filament\Actions;
+
+use Closure;
+use Filament\Actions\Imports\ImportColumn;
+use Filament\Actions\Imports\Importer;
+use Filament\Actions\Imports\Jobs\ImportCsv;
+use Filament\Actions\Imports\Models\Import;
+use Filament\Forms;
+use Filament\Forms\Components\Fieldset;
+use Filament\Forms\Components\FileUpload;
+use Filament\Forms\Components\Select;
+use Filament\Notifications\Actions\Action as NotificationAction;
+use Filament\Notifications\Notification;
+use Filament\Support\ChunkIterator;
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Filesystem\AwsS3V3Adapter;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\HtmlString;
+use League\Csv\Reader as CsvReader;
+use League\Csv\Statement;
+use League\Csv\Writer;
+use Livewire\Features\SupportFileUploads\TemporaryUploadedFile;
+use SplTempFileObject;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+use function Filament\Support\format_number;
+
+class ImportAction extends Action
+{
+    /**
+     * @var class-string<Importer>
+     */
+    protected string $importer;
+
+    protected ?string $job = null;
+
+    protected int | Closure $chunkSize = 100;
+
+    protected int | Closure | null $maxRows = null;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->label(fn (ImportAction $action): string => __('filament-actions::import.label', ['label' => $action->getPluralModelLabel()]));
+
+        $this->modalHeading(fn (ImportAction $action): string => __('filament-actions::import.modal.heading', ['label' => $action->getPluralModelLabel()]));
+
+        $this->modalDescription(fn (ImportAction $action): Htmlable => new HtmlString('<p>' . $action->getModalAction('downloadExample')->toHtml() . '</p>'));
```

Comment:
Why's this wrapped in `p` tags?

Reply:
You're gonna hate it but its to add some spacing, otherwise the button is too close to the heading. You can have a look when you pull it down if you want

Reply:
I checked, and the HTML was completely messed up.
`<p>` tags can't be nested, since that's invalid HTML.

I've removed the wrapping tag and the spacing looks good to me.
Consistent with other places like database notifications modal header actions.